Modules/unicodedata.c

```diff
@@ -1253,6 +1274,110 @@ unicodedata_UCD_lookup_impl(PyObject *self, const char *name,
     return PyUnicode_FromOrdinal(code);
 }
 
+typedef struct {
+    PyObject_HEAD
+    PyObject* str;
+    int pos;
+} GraphemeClusterIterator;
+
+void GCI_Del(PyObject* x)
```

Comment:
* The return type should be on separate line (see PEP 7).
* The function should be `static`.
* Usually destructors have the `dealloc` suffix.
* You can use the pointer to `GraphemeClusterIterator`.
* And it is more common to add a space before `*` than after it.
```
static void
GCI_dealloc(GraphemeClusterIterator *it)
```